### ISSUE TITLE
Add support for setting to run scripts in isolation

### ISSUE THREAD

**Issue by kimadeline** [microsoft/vscode-jupyter#4195]
We recently added a `python.useIsolation` setting in `vscode-python` to control running scripts in isolation: https://github.com/microsoft/vscode-python/pull/14845. Right now this setting only lives in the `isolate` branch of the `vscode-python` repo, but we're working towards adding support for this setting wherever we call the isolate script.

You may find it useful too 🙂

**Comment by DonJayamanne** [comment]
Details of why can be found here https://teams.microsoft.com/l/message/19:9b163ad1e1ed4c49938d9692be1ce751@thread.skype/1607553932217?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=6182cd1a-acf2-40e6-8b08-07bd25990df8&parentMessageId=1607553932217&teamName=Python%20for%20VS%20Code&channelName=Data%20Science&createdTime=1607553932217

TLDR - If users install packages using `pip -e`, then the only way to get stuff working for such packages is to ignore using the isolation setting.

**I believe this only applies to Jupyter extension when we start python kernels (or a damone that starts the python kernel)**

**Comment by IanMatthewHuff** [comment]
I believe this was deprecated by the trusted workspace work here: https://github.com/microsoft/vscode-python/pull/16485
